python/guide/packages/gluon/activations/activations.md

```diff
@@ -50,7 +51,7 @@ visualize_activation(mx.gluon.nn.Activation('sigmoid'))
 ```
 
 
-![png](images/sigmoid.png)
+![](images/sigmoid.png)
```

Comment:
Can we add a descriptive description? A plot of the sigmoid function?

Reply:
Same in other occurences

Reply:
Will add this to the QA checklist. Will remove undescriptive 'png' for now, and ask tutorial owners to add to their tutorials as required, as I don't think this has been done anywhere.